tron/config/schema.py

```diff
@@ -110,6 +110,7 @@ def config_object_factory(name, required=None, optional=None):
 ConfigMesos = config_object_factory(
     name='ConfigMesos',
     optional=[
+        'master_address',
```

Comment:
OK, can we either 1) make this required or 2) add validation that this exists if enabled is True?